pkgs/development/python-modules/deeplabcut/default.nix

```
@@ -0,0 +1,126 @@
+{ lib, buildPythonPackage, fetchPypi
+, python3
+, fetchzip
+, tensorflow
+, setuptools_scm
+, gzip
+, gnutar
+, certifi
+, chardet
+, click
+, easydict
+, h5py
+, imageio
+, imgaug
+, ipython
+, matplotlib
+, moviepy
+, numpy
+, opencv3
+, pandas
+, patsy
+, pillow
+, python
+, pyyaml
+, requests
+, ruamel_yaml
+, setuptools
+, scikitimage
+, scikitlearn
+, scipy
+, six
+, statsmodels
+, tables
+, tensorpack
+, tqdm
+, tkinter
+, wheel
+, wxPython_4_0
+}:
+
+buildPythonPackage rec {
+  pname = "deeplabcut";
+  version = "2.1.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b3c8c2a62d85c3fa188ecde04b473d38b9fba8e14537b4a88d4a82e61d0a1a2d";
+  };
+
+  resnet50 = fetchzip {
+    url = "http://download.tensorflow.org/models/resnet_v1_50_2016_08_28.tar.gz";
+    sha256 = "0n9k24ipgivvvd42h2bn26n4yjab0103ikjwhgcg668vbd5mv587";
+ };
+  resnet101 = fetchzip {
+    url = "http://download.tensorflow.org/models/resnet_v1_101_2016_08_28.tar.gz";
+    sha256 = "0h7b963px3xma936zysj54c8l930h3jyss7jv0gzbxalvxzq459x";
+ };
+
+  nativeBuildInputs = [ setuptools_scm  gzip gnutar];
```

Are you sure that you need gzip and gnutar?
```suggestion
  nativeBuildInputs = [ setuptools_scm  gzip gnutar ];
```

---

i don't thx